app/assets/javascripts/hyrax/relationships/confirm_remove_dialog.es6

```diff
@@ -0,0 +1,45 @@
+export default class ConfirmRemoveDialog {
+  /**
+   * The function to perform when the dialog is accepted.
+   *
+   * @callback requestCallback
+   */
+
+  /**
+   * Initialize the dialog
+   * @param {String} text the text for the body of the dialog
+   * @param {String} cancel the text for the cancel button
+   * @param {String} remove the text for the remove button
+   * @param {requestCallback} fn the function to perform if the remove button is pressed
+   */
+  constructor(text, cancel, remove, fn) {
+      this.text = text
+      this.cancel = cancel
+      this.remove = remove
+      this.fn = fn
+  }
+
+  template() {
+      return '<div class="modal confirm-remove-dialog" tabindex="-1" role="dialog">' +
```

Comment:
Is there a reason this mixes template literals and concatenation? It would be easier to read and change if the whole thing was a template literal. 

```javascript
return `<div class="modal confirm-remove-dialog" tabindex="-1" role="dialog"> 
        <div class="modal-dialog modal-md" role="document"> 
        <div class="modal-content"> 
        <div class="modal-body">${this.text}</div> 
        <div class="modal-footer"> 
          <button type="button" class="btn btn-default" data-dismiss="modal">${this.cancel}</button> 
          <button type="button" class="btn btn-danger" data-behavior="submit">${this.remove}</button> 
        </div> 
        </div> 
        </div> 
        </div>`
```

Reply:
You got it boss!